core/src/main/java/io/confluent/rest/Application.java

```diff
@@ -783,5 +790,27 @@ private void doShutdown() {
    */
   public void onShutdown() {
   }
+
+  /**
+   * Create the thread pool with request queue.
+   *
+   * @return thread pool used by the server
+   */
+  private ThreadPool createThreadPool() {
```

Comment:
I think maybe REQUEST_QUEUE_CAPACITY_INITIAL_DEFAULT should be like -1, so that it continues to use an UnboudedQueue, instead of a bounded queue with bound of INT_MAX.

Reply:
I think you mean REQUEST_QUEUE_CAPACITY_GROWBY_DEFAULT which controls how much capacity will be increased in need. The default value growby is Math.max(_minThreads, 8) in Jetty if user create Jetty server like new Server(). I put this default value to REQUEST_QUEUE_CAPACITY_GROWBY_DEFAULT = 64. I think we should be fine. 

The unbound queue will be controlled by both maximum capacity and growby. Current default maximum capacity is REQUEST_QUEUE_CAPACITY_DEFAULT = Integer.MAX_VALUE is same as Jetty one. So the default queue is unbounded.